tasks/internal/request-and-upload-signature/request-and-upload-signature.yaml

```diff
@@ -18,16 +18,19 @@ spec:
   params:
     - description: A docker image of operator-pipeline-images for the steps to run in.
       name: pipeline_image
-      default: "quay.io/redhat-isv/operator-pipelines-images:released"
+      default: "quay.io/konflux-ci/release-service-utils:0cf1c7923de64484f8737c0def90b74fe866040f"
       type: string
-    - description: Manifest digest for the signed content, usually in the format sha256:xxx
-      name: manifest_digest
-      type: string
-    - description: Docker reference for the signed content, e.g. registry.redhat.io/redhat/community-operator-index:v4.9
-      name: reference
+    - description: |
+        List of space separated manifest digests for the signed content, usually in the format sha256:xxx
+        e.g. sha256:xxx sha256:yyy
```

Comment:
This doesn't really add any new info. It's is already in the previous line.

Reply:
ok, this line can be removed